### PR TITLE
PAYARA-1843 Improve descriptive text on JMX Monitoring config

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
jenkins test please

File: appserver/admingui/jmx-monitoring-plugin/src/main/resources/fish/payara/admingui/jmxmonitoring/Strings.properties [hunk 1]
```diff
@@ -38,22 +38,22 @@
...
+jmxmonitoring.tab.tabToolTip=JMX Monitoring Logging
 
 jmxmonitoring.Enabled=Enabled
-jmxmonitoring.EnabledHelp=Enable JMX Monitoring for Payara Server
+jmxmonitoring.EnabledHelp=Enable JMX Monitoring Logs for Payara Server
 jmxmonitoring.AMXEnabled=AMX Enabled
 jmxmonitoring.AMXEnabledHelp=Whether or not to boot AMX on startup
 jmxmonitoring.Dynamic=Dynamic
-jmxmonitoring.DynamicHelp=If checked JMX Monitoring will be started dynamically
+jmxmonitoring.DynamicHelp=If checked JMX Monitoring Logging be started dynamically
```
Comment (smillidge) [inline appserver/admingui/jmx-monitoring-plugin/src/main/resources/fish/payara/admingui/jmxmonitoring/Strings.properties]: f checked JMX Monitoring Logging will be started dynamically

**Comment by Cousjava** [comment]
jenkins test please

**Comment by Cousjava** [comment]
replaced by #2098

**Pull request closed without merging**